torch/nn/modules/conv.py

```diff
@@ -986,6 +1029,7 @@ def forward(self, input: Tensor, output_size: Optional[List[int]] = None) -> Ten
         if self.padding_mode != 'zeros':
             raise ValueError('Only `zeros` padding mode is supported for ConvTranspose3d')
 
+        assert isinstance(self.padding, tuple)
```

Comment:
I think this assertion is wrong. The following code will crash because padding is `list`, not `tuple`

```python
import torch

x = torch.randn(4, 8, 32, 32, device='cuda', dtype=torch.float)
net = torch.nn.ConvTranspose2d(8, 16, kernel_size=3, padding=[3, 3])
net = net.cuda()

y = net(x)
torch.cuda.synchronize()
```

error message:

```
Traceback (most recent call last):
  File "/home/xwang/Developer/pytorch-test/random-things/padding-tuple.py", line 7, in <module>
    y = net(x)
  File "/home/xwang/Developer/pytorch/torch/nn/modules/module.py", line 1015, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/xwang/Developer/pytorch/torch/nn/modules/conv.py", line 890, in forward
    assert isinstance(self.padding, tuple)
AssertionError
```

Can you please fix it? @peterbell10

Reply:
@xwang233 thanks for pointing out! Can you please file a separate issue?